src/plugins/auto/plugin_config.cpp

```diff
@@ -39,6 +38,7 @@ void PluginConfig::set_default() {
         std::make_tuple(ov::device::full_name),
         std::make_tuple(ov::device::capabilities),
         std::make_tuple(ov::supported_properties));
+    devices_supported_properties = {CONFIG_KEY(CACHE_DIR)};
```

Comment:
why do we still need to handle cache_dir?
the idea is totally remove knowledge about non-auto plugin properties from AUTO plugin

Reply:
yes we don't need this, please remove

Reply:
I guess it's for case purpose? 
if yes, we can pass the cache_dir by compile model, also , may be the mock plugin should be updated to follow the new core<->plugin communication protocal.

Reply:
Because there are some test case to check compile_modle with incorrect property, if auto pass the incorrect to actual device, these test case will fail.

Reply:
> 

then these tests should be removed due to behavior change.  we can compile_model with any property, auto will check if it's also supported property, if yes, check the validity of property, if no, in parseMetaDevices() it will call core->get_supported_config to filter out config to hardware device

Reply:
Removed.
And update the test case `smoke_Auto_BehaviorTests, OVCompiledModelPropertiesIncorrectTests`, because there may have GPU in devices list when set 'AUTO', but GPU will not throw exception when num_streams is -100.